src/mappings/JoinAndQuit/schema.graphql

```diff
@@ -0,0 +1,9 @@
+type JoinAndQuitProposal @entity {
+	id: ID!
+	dao: DAO!
+	proposedMember: Bytes!
+	funder: Bytes!
+	funding: BigInt!
+	executed: Boolean!
```

Comment:
might need addition filed to say if the dao accept or reject the proposal.

Reply:
We have that in the Proposal entity. We could also remove the executed I guess.